src/explorer.ts

```diff
@@ -195,6 +196,40 @@ class KubernetesResource implements KubernetesObject, ResourceNode {
     }
 }
 
+class KubernetesNodeFolder extends KubernetesResourceFolder {
+    constructor() {
+        super(kuberesources.allKinds.node);
+    }
+
+    async getChildren(kubectl: Kubectl, host: Host): Promise<KubernetesObject[]> {
+        const nodes = await kubectlUtils.getGlobalResources(kubectl, 'nodes');
+        return nodes.map((node) => new KubernetesNodeResource(node.name, node));
+    }
+}
+
+class KubernetesNodeResource extends KubernetesResource {
+    constructor(name: string, meta: any) {
+        super(kuberesources.allKinds.node, name, meta);
+    }
+
+    async getTreeItem(): Promise<vscode.TreeItem> {
+        const treeItem = await super.getTreeItem();
+        treeItem.collapsibleState = vscode.TreeItemCollapsibleState.Collapsed;
+        return treeItem;
+    }
+
+    async getChildren(kubectl: Kubectl, host: Host): Promise<KubernetesObject[]> {
+        const pods = await kubectlUtils.getPods(kubectl, null, 'all');
+        const filteredPods = [];
+        pods.map((p) => {
+            if ('node/' + p.nodeName === this.resourceId) {
+                filteredPods.push(p);
+            }
+        });
+        return filteredPods.map((p) => new KubernetesResource(kuberesources.allKinds.pod, p.name, p));
```

Comment:
The `get` command fetches pods from all namespaces, but the existing `KubernetesResource` class doesn't consider the namespace when loading the resource.  We'll need to add this I think...

Reply:
Yeah, I think that is the bug I was seeing...